README.md

```diff
@@ -40,8 +40,38 @@ npm install
 
 ### Frontend microservices
 
-Currently unsupported.
-
+We offer a few ways of incorporating the GOV UK and HMRC CSS and JS libraries into your service. Pick the one which best suits your service.
+
+### 1. CDN
+If you are just using the GOVUK Frontend and HMRC Frontend and are not going to be adding any service-specific CSS or JS we suggest you use the HMRC CDN version of the assets.
+You will need to link to these in your service’s page template.
+
+1. add CDN js links to your `@bodyEnd` block in the layout template
+    - `<script type="text/javascript" src="https://www.tax.service.gov.uk/assets/hmrc-frontend/3.X.X/hmrc-frontend-3.X.X.min.js"></script>`
```

Comment:
Instead of `hmrc-frontend` I think this should be `govuk-frontend` as:

 - It's a more common use-case
 - The version 3.X.X seems to be referring to govuk-frontend

If you want to use `hmrc-frontend` it's best to change the version to `1.X.X`

It's probably worth explaining that `X` is a placeholder and linking to the list of versions in github.

Reply:
Good spot, 2 mins